docs/iris/src/developers_guide/documenting/whats_new_contributions.rst

```diff
@@ -0,0 +1,38 @@
+=================================
+Contributing a "What's New" entry
+=================================
+
+Iris has an aggregator for building an initial "What's New" document for each release, from sections contributed by code authors. This enables the release documentation to be written by the developer most familiar with the change while permitting editing to take place.
+
+Each release should have a directory, "contributions_<versionnumber>". Into this directory can be placed items for inclusion into the release documentation. The filename should be structured as follows:
+
+<category>_<date>_<summary>.txt
+
+The category will be one of the following:
+
+bugfix
+  A bugfix.
+newfeature
+  Features that are new or changed to add functionality.
+incompatiblechange
+  A change that causes an incompatibility with prior versions of Iris.
+deprecate
+  Deprecations of functionality.
+docchange
+  Changes to documentation.
+
+The date will be in the format of: a four digit year, two digit month, two digit day - separated by hyphens.
+
+Examples:
+* 2012-01-30
+* 2014-05-03
+* 2015-03-19
+
+The summary should be short, separated with hyphens.
+
+Examples:
+* whats-new-aggregator
+* netcdf-streaming
+* correction-to-bilinear-regrid
+
+Running make.py will check for the existence of a '<release>.rst' file for each contributions directory, and if none exists it will create one by aggregating the individual contributions.
```

Comment:
Agreed - I went with 'make.py' simply because MatPlotLib used the name for part of their document building. collate_contributions.py?